python/pyspark/sql/tests.py

```diff
@@ -3356,6 +3356,7 @@ def test_schema_conversion_roundtrip(self):
         self.assertEquals(self.schema, schema_rt)
 
 
+@unittest.skipIf(not _have_pandas or not _have_arrow, "Pandas or Arrow not installed")
```

Comment:
@ueshin @HyukjinKwon just confirming that this test should be conditional on pandas/pyarrow being installed as we will check for a minimum pyarrow version when using `pandas_udf `?

Reply:
I can't take a closer look now but let's do this if it passes the tests cc @ueshin

Reply:
Sorry for the delay. Yeah, we should check the minimum pyarrow version.